frontend/src/Todo/Body/BodyItemActions.jsx

```diff
@@ -0,0 +1,18 @@
+import { useContext } from 'react';
+import classes from './BodyItemActions.module.css';
+import { Context } from '../Context';
+
+export default function TodoBodyItemActions({ id }) 
+{
+	const { deleteBtn } = classes;
+
+	const { deleteTodo } = useContext(Context);
+
+	return (
+		<div>
+			<button onClick={() => deleteTodo(id)} className={deleteBtn}>
+        Delete
```

Comment:
this Delete either put him together with button tag like <button>Delete</button> or correct the indent

Reply:
done